sal/class.json-api-site-base.php

```diff
@@ -108,7 +108,7 @@ public function get_quota() {
 	abstract protected function is_a8c_publication( $post_id );
 
 	public function is_automated_transfer() {
-		return false;
+		return apply_filters( 'rest_api_site_option_is_atomic', false );
```

Comment:
Let's correct three things here:
* please call this filter `jetpack_site_automated_transfer`;
* please add a doc block along the lines of this: https://github.com/Automattic/jetpack/pull/9945/files#diff-98f87ca6991a9c0e844fcb759b5880a6R583
* also, we probably need to send a site id along with the filtered value because the hook handler will not be able to know what site applies the filter.